opba-protocols/hbci-protocol/src/main/java/de/adorsys/opba/protocol/hbci/config/CustomizedSinglePaymentJob.java

```diff
@@ -0,0 +1,35 @@
+package de.adorsys.opba.protocol.hbci.config;
+
+import de.adorsys.multibanking.domain.request.TransactionRequest;
+import de.adorsys.multibanking.domain.transaction.SinglePayment;
+import de.adorsys.multibanking.hbci.job.SinglePaymentJob;
+import lombok.experimental.Delegate;
+import org.kapott.hbci.GV.AbstractHBCIJob;
+import org.kapott.hbci.passport.PinTanPassport;
+
+@SuppressWarnings({"PMD.UnusedPrivateField"})
+public class CustomizedSinglePaymentJob extends SinglePaymentJob {
```

Comment:
I think we need to add a comment why this one exists or drop it completely (as it is just to set e2e id)